design/cluster-scope-resource-filter.md

```diff
@@ -0,0 +1,394 @@
+This is not allowed. Backup or restore cannot contain no resource.
+
+#### no namespaced resources + some cluster resources
+The following command means backup no namespaced resources and some cluster resources.
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=*
+	--include-cluster-scope-resources=StorageClass
+```
+
+#### no namespaced resources + all cluster resources
+The following command means backup no namespaced resources and all cluster resources.
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=*
+	--include-cluster-scope-resources=*
+```
+
+#### some namespaced resources + no cluster resources
+The following commands mean backup all resources in namespaces default and kube-system, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--exclude-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-cluster-resources=false
+```
+
+The following commands mean backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in all namespaces, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--exclude-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=false
+```
+
+The following commands mean backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in namespace default and kube-system, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--exclude-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=false
+```
+
+The following commands mean backup all resources except Ingress type resources in all namespaces, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=Ingress
+	--exclude-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-resources=Ingress
+	--include-cluster-resources=false
+```
+
+#### some namespaced resources + only related cluster resources
+This means backup all resources in namespaces default and kube-system, and related cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+```
+
+The following commands mean backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in all namespaces, and related cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
```

Comment:
But in this case resource like `pvc` or `cr` will be excluded right, therefore will there be any related cluster resources returned by BIA?

Reply:
Add PVC into the example.